### PR TITLE
Add unit tests for PlayUntilDone

### PR DESCRIPTION
**Pull request by andrecardoso**
- Small adjustments to make it easier to test

### CODE REVIEW

File: src/rise-play-until-done.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ RisePlayerConfiguration.PlayUntilDone = (() => {
         }
 
         if ( doneElements.length === playUntilDoneElements.length ) {
-          reportDone();
+          RisePlayerConfiguration.PlayUntilDone.reportTemplateDone();
```
Comment (olegrise) [inline src/rise-play-until-done.js]: @andrecardoso curious, what is the reason for adding `RisePlayerConfiguration.PlayUntilDone.`?

Comment (andrecardoso) [reply src/rise-play-until-done.js]: to mock it here: https://github.com/Rise-Vision/common-template/pull/69/files#diff-8289161080b1aa8a12980275c1b0f39cR59

Comment (olegrise) [reply src/rise-play-until-done.js]: @andrecardoso it seems that `this.reportTemplateDone();` solves the problem with mock. I think it's better than using external vs internal reference.

Comment (andrecardoso) [reply src/rise-play-until-done.js]: Changed. Thanks!

**Pull request merged**